src/OrchardCore.Modules/OrchardCore.Templates/Views/Template/Create.cshtml

```diff
@@ -17,7 +17,14 @@
 <script asp-src="~/OrchardCore.Liquid/codemirror/liquid.js" at="Foot"></script>
 <script asp-name="templatepreview-edit" at="Foot"></script>
 
-<zone Name="Title"><h1>@RenderTitleSegments(T["Create Template"])</h1></zone>
+@if (Model.AdminTemplates)
+{
+    <zone Name="Title"><h1>@RenderTitleSegments(T["Create Admin Template"])</h1></zone>
```

Comment:
Is it for Edit or Display templates or both?

Reply:
Fixing a tiny bug that I found in Templates module.  It's for `Create`

If `AdminTemplates` is true,  It should say 'Create Admin Template'